src/main/java/it/pagopa/transactions/services/TransactionsService.java

```diff
@@ -537,13 +537,13 @@ private Mono<NewTransactionResponseDto> projectActivationEvent(
                                 .authToken(sessionDataDto.getSessionToken())
                                 .status(TransactionStatusDto.fromValue(transaction.getStatus().toString()))
                                 // .feeTotal()//TODO da dove prendere le fees?
-                                .origin(
-                                        Optional.ofNullable(transaction.getOriginType())
+                                .clientId(
+                                        Optional.ofNullable(transaction.getClientId())
                                                 .map(
-                                                        origin -> NewTransactionResponseDto.OriginEnum
-                                                                .fromValue(origin.toString())
+                                                        clientId -> NewTransactionResponseDto.ClientIdEnum
+                                                                .fromValue(clientId.toString())
```

Comment:
The autogenerated `fromValue` method either returns an enum value or throws `IllegalArgumentException` (same behaviour as `valueOf`

```suggestion
                                                    .flatMap(
                                                        clientId -> {
                                                            try {
                                                                return Optional.of(NewTransactionResponseDto.ClientIdEnum.fromValue(clientId.toString()));
                                                            catch (IllegalArgumentException e) {
                                                                return Optional.empty();
                                                            }
                                                        }
                                                    )
```